mutacc_auto/parse/parse_scout.py

```diff
@@ -2,6 +2,7 @@
 from datetime import datetime, timedelta
 
 from mutacc_auto.commands.scout_command import ScoutExportCases
+from mutacc_auto.parse.vcf_constants import *
```

Comment:
Does this mean import all? If so, it is usually better to be explicit. Otherwise your namespace might be polluted without you knowing it.